templates/migration-wrapper.sql

```diff
@@ -7,7 +7,11 @@ DECLARE
     _body_differs BOOL = false;
     _migration _ps_catalog.migration = row ('{{filename}}', '{{version}}');
     _body TEXT = $migrationbody${{body}}$migrationbody$;
+    _old_search_path text;
 BEGIN
+    EXECUTE 'SHOW search_path' INTO STRICT _old_search_path;
+    SET search_path TO pg_catalog;
```

Comment:
we should just be able to use SET local here and avoid having to restore

Reply:
as above